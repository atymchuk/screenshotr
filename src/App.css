.App {
  text-align: center;
  /* background-image: url('./logo.svg'); */
  background-repeat: no-repeat;
  background-position: left;
}

.Logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.Header {
  background-color: #0a5f79;
  height: 150px;
  padding: 20px;
  color: #e2e2e2;
}

.Title {
  font-size: 1.5em;
}

.Intro {
  font-size: large;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}
