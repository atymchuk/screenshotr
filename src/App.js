import React, { Component } from 'react';
import logo from './logo.svg';
import styles from './App.css';

class App extends Component {
  render() {
    return (
      <div className={styles.App}>
        <header className={styles.Header}>
          <img src={logo} className={styles.Logo} alt="logo" />
          <h1 className={styles.Title}>Welcome to React</h1>
        </header>
        <p className={styles.Intro}>
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
