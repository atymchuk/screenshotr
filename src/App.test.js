import React from 'react';
import ReactDOMServer from 'react-dom/server';
import renderer from 'react-test-renderer';
import App from './App';

describe('test App component', () => {

  it('renders correctly, matching the snapshot', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders with SSR', () => {
    const html = ReactDOMServer.renderToStaticMarkup(<App />);
    console.log(html);
  });

});
