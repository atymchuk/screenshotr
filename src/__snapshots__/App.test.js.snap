// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App component renders correctly, matching the snapshot 1`] = `
<div
  className="App__App___1o-Fp"
>
  <header
    className="App__Header___3mO0c"
  >
    <img
      alt="logo"
      className="App__Logo___2xoEc"
      src="logo.svg"
    />
    <h1
      className="App__Title___2x147"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="App__Intro___X9pKe"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
